src/components/LoanCards/LoanCardImage.vue

```diff
@@ -90,14 +90,15 @@ export default {
 	methods: {
 		handleImageClick(event) {
 			this.$emit('image-click');
-			if (this.disableLink) {
-				event.preventDefault();
-				return;
-			}
+
 			this.$emit('track-loan-card-interaction', {
```

Comment:
This was previously not getting fired with the disableLink case. It never affected anything in prod.